src/components/HeaderInfo/HeaderInfo.test.js

```diff
@@ -1,12 +1,12 @@
-// import React from "react";
-// import {
-//   render,
-//   screen,
-//   within,
-//   fireEvent,
-//   waitForElement,
-// } from "@testing-library/react";
-// import HeaderInfo from "./HeaderInfo";
+import React from "react";
+import {
+  render,
+  screen,
+  within,
+  fireEvent,
+  waitForElement,
+} from "@testing-library/react";
+import HeaderInfo from "./HeaderInfo";
 // // import { getActiveConfigurations } from "../../utils/selectors/monitoringConfigurations";
```

Comment:
please remove commented out content if not using

Reply:
Used as reference for pending updates to unit tests